daml-script/daml/Daml/Script.daml

```diff
@@ -731,3 +749,127 @@ submitTreeMulti actAs readAs cmds =
     commands = void cmds
     locations = getCallStack callStack
     continue = identity
+
+-- | HIDE User as used in the user management service
+data User = User
+  with
+    id : Text
```

Comment:
I’m slightly unsure about this. we could also newtype this but I’m not sure that really leads to a nicer API. With done that with the party id hint and it always looked awkwards.